assets/js/common/OperationModals/OperationModal.jsx

```diff
@@ -0,0 +1,71 @@
+import React from 'react';
+import { EOS_WARNING_OUTLINED } from 'eos-icons-react';
+
+import Modal from '@common/Modal';
+import Button from '@common/Button';
+import Input from '@common/Input';
+
+function OperationModal({
+  title,
+  description,
+  operationText,
+  applyDisabled = true,
+  checked = false,
+  isOpen = false,
+  onChecked,
```

Comment:
We could import lodash noop. 
For onChecked, onRequest and onCancel

Reply:
Done!